cloud-optimized-geotiffs/writing-cogs-in-python.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<p>Label the axis, I have no idea what I'm looking at. See <a href="https://docs.maap-project.org/en/latest/technical_tutorials/access/accessing_cod.html#Accessing-Cloud-Optimized-Data" target="_blank">https://docs.maap-project.org/en/latest/technical_tutorials/access/accessing_cod.html#Accessing-Cloud-Optimized-Data</a> for what a DEM map should look like.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>